aspnetcore/security/key-vault-configuration.md

```diff
@@ -210,25 +210,24 @@ config.AddAzureKeyVault(
 
 | Property         | Description |
 | ---------------- | ----------- |
-| `Client`         | <xref:Microsoft.Azure.KeyVault.KeyVaultClient> to use for retrieving values. |
-| `Manager`        | <xref:Microsoft.Extensions.Configuration.AzureKeyVault.IKeyVaultSecretManager> instance used to control secret loading. |
+| `Client`         | <xref:Azure.Security.KeyVault.Secrets.SecretClient> to use for retrieving values. |
```

Comment:
This property is not part of  `AzureKeyVaultConfigurationOptions` in track 2. https://docs.microsoft.com/en-us/dotnet/api/azure.extensions.aspnetcore.configuration.secrets.azurekeyvaultconfigurationoptions?view=azure-dotnet